app/controllers/publishers/vacancies/documents_controller.rb

```diff
@@ -3,15 +3,15 @@
 class Publishers::Vacancies::DocumentsController < Publishers::Vacancies::BaseController
   helper_method :form
 
-  before_action :redirect_to_next_step, only: %i[create]
+  before_action :complete_step, unless: :document_added?, only: %i[create]
 
   def create
     form.valid_documents.each do |document|
       vacancy.supporting_documents.attach(document)
       send_event(:supporting_document_created, document.original_filename, document.size, document.content_type)
     end
 
-    render :show
+    redirect_to organisation_job_build_path(vacancy.id, :documents, back_to_review: params[:publishers_job_listing_documents_form][:back_to_review])
```

Comment:
not sure I understand this line

Reply:
I guess you redirect to preserve params[:back_to_review]

Reply:
Yeah. Have a since changed this cos it meant that errors weren't be displayed